guardrails/ingestion_service.py

```diff
@@ -0,0 +1,88 @@
+import os
+import hashlib
+from guardrails.document_store import Document, DocumentStoreBase
+from guard_rails_api_client import AuthenticatedClient
+from typing import Any, Dict, List, Optional
+from guard_rails_api_client.api.default import ingest
+from guard_rails_api_client.models import IngestionPayload
+
+try:
+    import sqlalchemy
+    import sqlalchemy.orm as orm
+except ImportError:
+    sqlalchemy = None
+    orm = None
+
+
+class IngestionServiceDocumentStore(DocumentStoreBase):
+    def __init__(self, base_url: str = None, api_key: str = None):
+        self.base_url = (
+            base_url
+            if base_url is not None
+            else os.environ.get("GUARDRAILS_BASE_URL", "http://localhost:5000") # TODO: switch the default from localhost to our hosted endpoint
```

Comment:
We'll need to have a different env var for this base url vs the crud api's base url.